include/z64actor.h

```diff
@@ -65,103 +62,131 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ u8 health;
-    /* 0x02 */ s16 unk_02;
-    /* 0x04 */ s16 unk_04;
+    /* 0x02 */ s16 cylRadius;
+    /* 0x04 */ s16 cylHeight;
     /* 0x06 */ u8 mass;
 } CollisionCheckInfoInit;
 
 typedef struct {
     /* 0x00 */ u8 health;
-    /* 0x02 */ s16 unk_02;
-    /* 0x04 */ s16 unk_04;
-    /* 0x06 */ s16 unk_06;
+    /* 0x02 */ s16 cylRadius;
+    /* 0x04 */ s16 cylHeight;
+    /* 0x06 */ s16 cylYShift;
     /* 0x08 */ u8 mass;
 } CollisionCheckInfoInit2;
 
 typedef struct {
-    /* 0x00 */ DamageTable* damageTable;  // For actors which contain one (example: Stalfos)...
-    /* 0x04 */ Vec3f displacement; // Amount to correct velocity (0x5C) by when colliding into a body
-    /* 0x10 */ s16   unk_10;
-    /* 0x12 */ s16   unk_12;
-    /* 0x14 */ u16   unk_14;
-    /* 0x16 */ u8    mass; // Used to compute displacement, 50 is common value, 0xFF for infinite mass/unmoveable
-    /* 0x17 */ u8    health;
-    /* 0x18 */ u8    damage; // Amount to decrement health by
-    /* 0x19 */ u8    damageEffect; // Stores what effect should occur when hit by a weapon
-    /* 0x1A */ u8    atHitEffect; // Stores what effect should occur when AT connects with an AC
-    /* 0x1B */ u8    acHitEffect; // Stores what effect should occur when AC is touched by an AT
+    /* 0x00 */ DamageTable* damageTable;
+    /* 0x04 */ Vec3f displacement; // Amount to correct velocity (0x5C) by when colliding into a body.
+    /* 0x10 */ s16 cylRadius; // Used for various purposes.
+    /* 0x12 */ s16 cylHeight; // Used for various purposes.
+    /* 0x14 */ s16 cylYShift; // Unused. Intended purpose inferred from the Cylinder16 structure and func_80062ECC.
```

Comment:
```suggestion
    /* 0x14 */ s16 cylYShift; // Unused. Intended purpose inferred from the Cylinder16 structure and CollisionCheck_CylSideVsLineSeg.
```

Reply:
dang this makes the comment too long and puts it on the next line lmfao. might remove the mention of the function alltogether

Reply:
got it to work